client/state/ui/language/reducer.js

```diff
@@ -17,7 +17,7 @@ import { localeSlugSchema } from './schema';
  *
  */
 export const localeSlug = createReducer(
-	false,
```

Comment:
I don't follow why we're changing the default value here.

Reply:
It's just demonstrating that this is clearly invalid according to the schema, but validated. I don't believe this change is intended to merge.

Reply:
Did you mean to update this to `null`?

Reply:
Right now we can't write said unit test because it will fail when it should pass (or however you look at it).